_layouts/default.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang=fr>
+	<head>
+		<meta charset="utf-8">
+		<title>AgileFrance</title>
+
+		<link href="css/main.css" rel="stylesheet" type="text/css" media="all">
+	</head>
+	<body>
+		<header>
+			<h1><a href="/">Association Agile France</a></h1>
+			<img src="/logo.svg" height=150 alt=Logo />
```

Comment:
This is not a proper use of `alt`. “Logo” is not descriptive of the contents of the image. The proper generic use should be to describe in `alt` what the image shows, which is “Association Agile France”.
However, since this is already written as text right before, the best is to simply specify an empty `alt` attribute, as the image is illustrative only.

To determine what is best to write in an alt, you can simply read in your head the text that comes before the `<img>`, then the contents of `alt`, then the text that comes after. Here, what sounds more appropriate for the `<header>` between:

1. “Association Agile France Logo”
2. “Association Agile France Association Agile France”
3. “Association Agile France”

For me it is 3, thus we should have `alt=""` 🙂

You can read more about `alt` in the [HTML spec](https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element) (see the examples at the bottom of the page) and get a couple tips [here](https://www.phase2technology.com/blog/no-more-excuses-definitive-guide-alt-text-field).

Reply:
[Fair enough](https://www.w3.org/WAI/tutorials/images/functional/#logo-image-within-link-text)